tests/core/full_node/test_mempool_performance.py

```diff
@@ -69,12 +70,22 @@ async def test_mempool_update_performance(self, bt, wallet_nodes_mempool_perf, d
         blocks = bt.get_consecutive_blocks(3, blocks)
         await full_node_api_1.full_node.respond_block(full_node_protocol.RespondBlock(blocks[-3]))
 
+        fast_durations: List[float] = []
+        slow_durations: List[float] = []
+        
         for idx, block in enumerate(blocks):
             start_t_2 = time.time()
             await full_node_api_1.full_node.respond_block(full_node_protocol.RespondBlock(block))
             end_t_2 = time.time()
             duration = end_t_2 - start_t_2
             if idx >= len(blocks) - 3:
+                slow_durations.append(duration)
                 assert duration < 0.1
             else:
+                fast_durations.append(duration)
                 assert duration < 0.0003
+
+        fast_mean = mean(fast_durations)
+        slow_mean = mean(slow_durations)
+
+        assert fast_mean < 0.0002 and slow_mean < 0.1
```

Comment:
0.0002 is really fast, i would change it to something like 0.001, that is fast enough anyway

Reply:
Chatting with Arvid about this as well.  I was just putting it back where it was before the bump to 0.0003.  I was hoping that the averaging would suffice for the moment.